linters/import/README.md

```diff
@@ -0,0 +1,53 @@
+# Import
```

Comment:
`import` as a cli tool name is a little vague, can you rename to make it clearer what it does. e.g. `importorder`

Reply:
Also, could you add a reference to the README on the main page for the new linter: https://github.com/m3db/build-tools/blob/master/README.md